core/types/imports.go

```diff
@@ -46,12 +49,17 @@ const (
 
 // The following functions are used directly as their upstream definitions.
 var (
-	BloomLookup         = ethtypes.BloomLookup
-	BytesToBloom        = ethtypes.BytesToBloom
-	CreateBloom         = ethtypes.CreateBloom
-	NewReceipt          = ethtypes.NewReceipt
-	NewContractCreation = ethtypes.NewContractCreation
-	NewTransaction      = ethtypes.NewTransaction
+	BloomLookup          = ethtypes.BloomLookup
+	BytesToBloom         = ethtypes.BytesToBloom
+	CreateBloom          = ethtypes.CreateBloom
+	NewReceipt           = ethtypes.NewReceipt
+	NewContractCreation  = ethtypes.NewContractCreation
+	NewTransaction       = ethtypes.NewTransaction
+	EncodeNonce          = ethtypes.EncodeNonce
+	NewEmptyStateAccount = ethtypes.NewEmptyStateAccount
+	SlimAccountRLP       = ethtypes.SlimAccountRLP
+	FullAccount          = ethtypes.FullAccount
+	FullAccountRLP       = ethtypes.FullAccountRLP
```

Comment:
We might want to sort these (and existing ones too) alphabetically to have a consistent order with subnet-evm as well.

Reply:
Done in #800